### ISSUE TITLE
Set up Sphinx docs?

### ISSUE THREAD

**Issue by kyleabeauchamp** [rmcgibbo/openmm-cmd#33]
It might be cute to have an online HTML doc that nicely groups all the different command line options that are available.  This could even be done in an automated way.

**Comment by leeping** [comment]
That would indeed be nice.  It could be a good exercise for me in writing Sphinx documentation, question is how soon I will get to it! :)

BTW I'm currently using the command line to run our T4 lysozyme simulations.

**Comment by kyleabeauchamp** [comment]
Can you resume a trajectory and append to the same DCD file?

**Comment by leeping** [comment]
It currently backs up the file.  Are DCD files trivially appendable?  My impression was that the OpenMM DCDReporter overwrites the file if it exists.

**Comment by kyleabeauchamp** [comment]
See https://github.com/rmcgibbo/mdtraj/issues/441

**Comment by leeping** [comment]
Also, it is possible to resume trajectories.  I'm saving the restart information to a JSON format and then compressing it.

**Comment by kyleabeauchamp** [comment]
So it resumes, but creates a new DCD file?

**Comment by leeping** [comment]
Yeah, that's right.  If the DCD file being written already exists, the existing file is backed up to `filename_#.dcd` where `#` is the smallest number corresponding to a file name that doesn't exist.

**Comment by leeping** [comment]
The restart file is backed up as well.

Side notes: On a Titan I am getting 55 ns/day with your provided system.  In order to convert your PDB to a Gromacs .top file using "pdbgmx", I had to rename several hydrogens and residue names "HIS" -> "HID".  OpenMM then loads the Gromacs .gro and .top file without any issues.  In the future I will convert the my Gromacs .itp files to OpenMM .xml files, which means we won't need to load Gromacs coordinates and topologies anymore.
